src/api/app/components/build_result_for_architecture_component.rb

```diff
@@ -0,0 +1,68 @@
+class BuildResultForArchitectureComponent < ApplicationComponent
+  attr_accessor :result, :project, :package
+
+  def initialize(result, project, package)
+    super
+
+    @result = result
+    @project = project
+    @package = package
+  end
+
+  private
+
+  def result_code
+    result.code
+  end
+
+  def help
+    help = "<p><strong>Package build ( #{build_status_icon} #{result_code} ):</strong> #{build_status_help}</p>"
+    help += "<p><u>Details</u>: #{build_status_details}</p>" if build_status_details.present?
```

Comment:
What if we keep the `build_status_details` closer to the `build_status_help` part as they are related one each other? Additionally I'd uniform the `strong` style for the label instead of the `underline` there. Something like the following
![image](https://user-images.githubusercontent.com/7080830/202455092-f4115ca3-7b84-4af8-9ae5-9d865f1402d3.png)

Reply:
We need to find a better place and way to display the details, which can be long text.
We could even split the static description and the details which are specific.

I create a card for that: https://trello.com/c/jJgasnGs/57-separate-static-description-of-the-status-and-the-details

Reply:
Thanks for taking care of that. We will tackle this in detail later, no reason to be picky and stop the rest here. :+1: